### ISSUE TITLE
A11y_AzureDataStudio_ServiceSettings_Keyboard:Expand/collapse button is not accessible by the keyboard.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7601]
"[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:** #A11y_AzureDataStudio;#A11yMAS; #A11yTCS;#SQL Azure Data Studio;#WCAG2.1.1;#DesktopApp;#Keyboard; #Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Repro Steps:**

1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on
        select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button.
8.	Navigate on the "service setting" screen using tab navigation.
9.    Check expand/collapse button is actionable by the keyboard or not.

**Actual:**
Expand/collapse button is not accessible by the keyboard.

**Expected:**
Keyboard focus should go on expand/collapse button and accessible by the keyboard.

**Same issue observed for the below scenario:**
**1.Summary:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Enter password and confirm password and click on next button and again click on next button.
8.	Verify that all the elements of " Summary" are accessible and MAS Compliant.

**User Impact:**
Keyboard only user get difficulty to access the control.

**Attachment for Reference**
[7601_A11y_AzureDataStudio_ServiceSettings_Keyboard.pptx](https://github.com/microsoft/azuredatastudio/files/3711281/7601_A11y_AzureDataStudio_ServiceSettings_Keyboard.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by vinodbk197** [comment]
Verified the issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is resolved,Hence closed this issue.
Please refer the below attachment:
[7601_Closed.pptx](https://github.com/microsoft/azuredatastudio/files/4086384/7601_Closed.pptx)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;

**Comment by Yash14j** [comment]
#A11ysev2;#Closed;
